### PR TITLE
optional ViewContainerRef and RxJS-operators modal support added

### PR DESCRIPTION
**Pull request by askarby**
:children_crossing: Since ViewContainerRef was not needed anymore, passing it to the ModalController is now optional (will be removed in later version of Kirby). Made operators (to be used with RxJs) to use modals, alerts and action sheets in a reactive manor.

### CODE REVIEW

File: src/app/examples/alert-example/alert-example.component.ts [hunk 1]
```diff
@@ -2,25 +2,41 @@ import { Component, ViewContainerRef } from '@angular/core';
 
 import { ModalController } from '@kirbydesign/designsystem/modal';
 import { AlertConfig } from '@kirbydesign/designsystem/modal';
+import { of } from 'rxjs';
```
Comment (jonasjensen-brolstark) [inline src/app/examples/alert-example/alert-example.component.ts]: The linter is complaining about these imports. They need to be at top as they are 3. party.

Comment (askarby) [reply src/app/examples/alert-example/alert-example.component.ts]: Well, that was embarrassing... fixed!

**Comment by hafstad** [comment]
Also please label your pull-req according to which component you're changing, and mark it with "breaking change" if thats going to be the case. In that way our changelog will affect that when it's merged :)
https://cookbook.kirby.design/home/changelog

File: src/app/examples/alert-example/alert-example.component.ts (hunk 1 above)
Comment (askarby) [reply src/app/examples/alert-example/alert-example.component.ts]: In regards to the "2 second delay", I wanted to show the "cheapest possible solution" to having a RxJs Observable do something.... granted.... I could simply make it wait 10ms... that way the impact wouldn't be annoying.

**Comment by askarby** [comment]
Closing this in favor of: https://github.com/kirbydesign/designsystem/pull/610

**Pull request closed without merging**